pkg/karmadactl/get.go

```diff
@@ -58,6 +58,7 @@ var (
 	eventColumn = metav1.TableColumnDefinition{Name: "EVENT", Type: "string", Format: "", Priority: 0}
 
 	getShort = `Display one or many resources`
+	getLog   = `Display one or many resources.`
```

Comment:
`getLog` --> `getLong`

Reply:
@lonelyCZ Thank you for pointing out the error, it has been modified